admin_manual/configuration_server/activity_configuration.rst

```diff
@@ -50,6 +50,15 @@ like in normal shares when set to ``true``.
     2. Users that had access to a groupfolder, share or external storage can see activities in their stream and emails that happen after they are removed until they login again
     3. Users that are newly added to a groupfolder, share or external storage can not see activities in their stream nor emails that happen after they are added until they login again
 
+.. _label-activities-externalstorages:
+
+Activities in external storages
```

Comment:
Just combine with the section above in a common headline?
The 3rd note mentions them already

Reply:
alright, I saw it now. thanks for pointing out.

sections are combined